packages/core-database-sequelize/__tests__/repositories/transactions.test.js

```diff
@@ -99,6 +99,21 @@ describe('Transaction Repository', () => {
       })
     })
 
+    it('should find the same transactions with parameter senderId and corresponding senderPublicKey', async () => {
+      await connection.saveBlock(genesisBlock)
+      const senderWallet = await spv.walletManager.getWalletByPublicKey('034776bd6080a504b0f84f8d66b16af292dc253aa5f4be8b807746a82aa383bd3c')
+
+      const transactionsSenderPublicKey = await repository.findAll({ senderPublicKey: senderWallet.publicKey })
+      const transactionsSenderId = await repository.findAll({ senderId: senderWallet.address })
+
+      expect(transactionsSenderPublicKey.count).toBe(transactionsSenderId.count)
```

Comment:
The last change: check the number of rows too.

We know, from the implementation, that the number of rows and count should be the same, but it should be tested too. Currently iit's evident that they're the same because the implementation is very simple, but it could be changed in the future, so testing the number of rows would be the safer.

Reply:
Ok I understand. Check out my last commit !